modules/v2x/v2x_proxy/proto_adapter/proto_adapter.cc

```diff
@@ -0,0 +1,230 @@
+          res_single1_ptr->set_right_turn_light(obu_single1.right_turn_light());
+        }
+        if (obu_single1.has_color()) {
+          res_single1_ptr->set_color(obu_single1.color());
+        }
+        if (obu_single1.has_next_color()) {
+          res_single1_ptr->set_next_color(obu_single1.next_color());
+        }
+        if (obu_single1.has_traffic_light_type()) {
+          res_single1_ptr->add_traffic_light_type(
+              AdapterLightTypeObu2Sys(obu_single1.traffic_light_type()));
+        }
+      }
+    }
+  }
+  *os_light_ptr = res;
+  return flag_has_data;
+#endif
+  return false;
+}
+
+bool AdapterRsiObu2Sys(const OBURsi *obu_rsi,
+                       std::shared_ptr<OSRsi> *os_rsi) {
+  if (nullptr == obu_rsi) {
+    return false;
+  }
+  if (nullptr == os_rsi) {
+    return false;
+  }
+  auto res = std::make_shared<OSRsi>();
+  if (nullptr == res) {
+    return false;
+  }
+  if (!obu_rsi->has_alter_type()) {
+    return false;
+  }
+  res->set_radius(obu_rsi->radius());
+  res->set_rsi_type(obu_rsi->alter_type());
+  if (obu_rsi->has_header()) {
+    auto header = res->mutable_header();
+    header->set_sequence_num(obu_rsi->header().sequence_num());
+    header->set_timestamp_sec(obu_rsi->header().timestamp_sec());
+    header->set_module_name("v2x");
+  }
+  switch (obu_rsi->alter_type()) {
+    case RsiAlterType::SPEED_LIMIT:
+    case RsiAlterType::SPEED_LIMIT_BRIDGE:
+    case RsiAlterType::SPEED_LIMIT_TUNNEL:
+      for (int index = 0; index < obu_rsi->points_size(); index++) {
+        auto point = res->add_points();
+        point->set_x(obu_rsi->points(index).x());
+        point->set_y(obu_rsi->points(index).y());
+      }
+      res->set_speed(std::atof(obu_rsi->description().c_str()));
+      break;
+    case RsiAlterType::CONSTRUCTION_AHEAD:  // Construction Ahead
+    case RsiAlterType::BUS_LANE:            // Bus Lane
+    case RsiAlterType::TIDAL_LANE:          // tidal Lane
+    case RsiAlterType::TRAFFIC_JAM:         // traffic jam
+    case RsiAlterType::TRAFFIC_ACCIDENT:    // traffic accident
+      for (int index = 0; index < obu_rsi->points_size(); index++) {
+        auto point = res->add_points();
+        point->set_x(obu_rsi->points(index).x());
+        point->set_y(obu_rsi->points(index).y());
+      }
+      break;
+    case RsiAlterType::NO_HONKING:          // No Honking
+    case RsiAlterType::SLOW_DOWN_SECTION:   // slow down section
+    case RsiAlterType::ACCIDENT_PRONE:      // Accident Prone
+    case RsiAlterType::OVERSPEED_VEHICLE:   // overspeed vehicle
+    case RsiAlterType::EMERGENCY_BRAKING:   // emergency braking
+    case RsiAlterType::ANTIDROMIC_VEHICLE:  // antidromic vehicle
+    case RsiAlterType::ZOMBIES_VEHICLE:     // zombies vehicle
+    {
+      auto point = res->add_points();
+      point->set_x(obu_rsi->points(0).x());
+      point->set_y(obu_rsi->points(0).y());
+    } break;
+    case RsiAlterType::CONTROLLOSS_VEHICLE:  // controlloss vehicle
```

Comment:
Have you run `scripts/clang_format.sh <path/to/cpp/dirs/or/files>` or `./apollo.sh format <path/to/dirs/or/files>` ?

Reply:
Yes. If I add a '\n' here; Clang-format will remove it.

Reply:
Coding style
```
case A: {
}
```